### PR TITLE
Add another possible cause

### PR DESCRIPTION
**Pull request by SimpleSamples**
There seem to be many possible causes and for me I copied code that had the attribute name CodeFile that should be CodeBehind for my project. Fixing that removed many and all errors I was getting.

I know that my wording is inadequate but I assume someone can make it more appropriate.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by BillWagner** [comment]
@scottaddie or @RickAndMsft 

Can you review or suggest updated wording for this PR?

File: docs/csharp/language-reference/compiler-messages/cs0433.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ The type TypeName1 exists in both TypeName2 and TypeName3
  Two different assemblies referenced in your application contain the same namespace and type, which produces ambiguity.  
   
  To resolve this error, use the alias feature of the [/reference (C# Compiler Options)](../../../csharp/language-reference/compiler-options/reference-compiler-option.md) compiler option or do not reference one of your assemblies.  
+ 
+ This error can also occur if the Page tag has CodeFile when it should be CodeBehind.
+ 
+ This error can also occur if code is in an APP_CODE folder that should not be in an APP_CODE folder.
```
Comment (scottaddie) [inline docs/csharp/language-reference/compiler-messages/cs0433.md]:
```suggestion
```

File: docs/csharp/language-reference/compiler-messages/cs0433.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ The type TypeName1 exists in both TypeName2 and TypeName3
  Two different assemblies referenced in your application contain the same namespace and type, which produces ambiguity.  
   
  To resolve this error, use the alias feature of the [/reference (C# Compiler Options)](../../../csharp/language-reference/compiler-options/reference-compiler-option.md) compiler option or do not reference one of your assemblies.  
+ 
+ This error can also occur if the Page tag has CodeFile when it should be CodeBehind.
```
Comment (scottaddie) [inline docs/csharp/language-reference/compiler-messages/cs0433.md]:
```suggestion
This error can also occur in ASP.NET Web Forms if:

* The `@ Page` directive has a `CodeFile` attribute when it should be a `CodeBehind` attribute.
* Code is placed in an *App_Code* folder that shouldn't reside there.
```

**Review by scottaddie — DISMISSED** [review]

**Comment by BillWagner** [comment]
self-assigning to incorporate suggestions and merge.

**Comment by BillWagner** [comment]
@SimpleSamples I don't have the permissions to add Scott's suggestions to this PR. Can you commit those changes? Once that's done, I'll :shipit:

Or, if you allow edits on this PR from maintainers, I'll commit those changes and merge.

Thanks.

**Comment by SimpleSamples** [comment]
Okay, I think I did the commit.

**Review by BillWagner — APPROVED** [review]
After updates based on reviews, this is ready to :shipit:

Thanks again @SimpleSamples

**Pull request merged**